### PR TITLE
[QA] Check security_opt setting

### PR DESCRIPTION
**Pull request by saw-jan**

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/12774/2/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/12774/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/12774/guiReportUpload/serverlog.log

Screenshots:
- [logging_out.png](https://cache.owncloud.com/public/owncloud/client/12774/guiReportUpload/screenshots/logging_out.png)

</details>

File: .drone.star [hunk 1]
```diff
@@ -265,6 +263,7 @@ def gui_tests(squish_parameters = "", depends_on = []):
             "SQUISH_PARAMETERS": squish_parameters,
             "STACKTRACE_FILE": STACKTRACE_FILE,
         },
+        "security_opt": "seccomp=unconfined",
```
Comment (fmoc) [inline .drone.star:266]: What problem do you want to solve by removing the confinement, i.e., vastly decreasing the security of the CI pipeline?

**Review by fmoc — CHANGES_REQUESTED** [review]

File: .drone.star (hunk 1 above)
Comment (saw-jan) [reply .drone.star:266]: Yeah, I was just trying check if drone would support that or not. I would not implement that anyway but that was the only workaround to make xfce4 of latest squish image work on ubuntu20.04 host (on my local system)

**Comment by saw-jan** [comment]
This was just only the quick check of function availability. Seems like drone would not allow changes in security settings.
Closing here!

**Pull request closed without merging**